### PR TITLE
Fix value of iconBrandColor for independent connectors

### PR DESCRIPTION
**Pull request by vbossica**
According to the guidelines, the iconBrandColor for independent connectors must be `#da3b01`.

### CODE REVIEW

File: independent-publisher-connectors/Brønnøysundregistrene/apiProperties.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {
   "properties": {
     "connectionParameters": {},
-    "iconBrandColor": "#0B576F",
+    "iconBrandColor": "#da3b01",
     "capabilities": [],
     "publisher": "Ahmad Najjar",
-    "stackOwner": "Br�nn�ysundregistrene (Br�nn�ysund registries)"
+    "stackOwner": "Brønnøysundregistrene (Brønnøysund registries)"
```
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/Brønnøysundregistrene/apiProperties.json:7]: @vbossica we don't allow any special characters. can you please revert this change? Thank you

**Review by Amjed-Ayoub — CHANGES_REQUESTED** [review]

**Comment by Amjed-Ayoub** [comment]
@vbossica thank you so much for your help with this fix we really appreciate it.

File: independent-publisher-connectors/Brønnøysundregistrene/apiProperties.json (hunk 1 above)
Comment (vbossica) [reply independent-publisher-connectors/Brønnøysundregistrene/apiProperties.json]: Replaced the `ø` with `o` in the last commit

**Review by Amjed-Ayoub — APPROVED** [review]
